target/bin/updatemailuser

```diff
@@ -27,8 +27,9 @@ fi
 
 HASH="$(doveadm pw -s SHA512-CRYPT -u "${USER}" -p "${PASSWD}")"
 
-# Protect config file with lock to avoid race conditions
 touch "${DATABASE}"
-create_lock "$(basename "$0")"
+# Protect config file with lock to avoid race conditions
+LOCK_ID="$(uuid)"
```

Comment:
I assume `LOCK_ID` is no longer meant to be here and was missed for removal?

Reply:
100%